src/frameworks/foundation/ns_util.rs

```diff
@@ -0,0 +1,24 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+//! Functions under `Objective-C Runtime Utilities`, just `NSStringFromClass` right now.
+
+use crate::{
```

Comment:
Stylistic nit: For consistency with the rest of the code, could you use the pattern `use crate::module::item;`/`use crate::module::{item1, item2};`? There's nothing really wrong with this style, it's just quite different from all the rest of the code.

Also, uh, please import `export_c_func` from `dyld` rather than the crate root. It's only exported from the latter due to scoping quirks.

Reply:
I'm going to be honest, I pretty much just used my LSP auto-import for pretty much all of the imports. Most of the other ones I'll need to clean up manually, but for this one there's actually a rustfmt lint, so I'll add it in and you can merge if you want.

Reply:
Ah I see. If there's some way I can help LSPs to match what I do manually (or vice-versa), let me know.

Reply:
Just realized it's unstable :P. Probably won't add it then,

Reply:
All the good lints are :(